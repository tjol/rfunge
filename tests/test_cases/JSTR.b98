"RTSJ"4("et1i a ucs"a02-047*3bP"secs ":"s ":"st"2073bPv@

        a
    >v"XXXXXXXXXXXXXXXXXXXXXX"0                       <
    ,:
    ^_$01-vi
          : s
          >8b +v
              td
            >,ve
            |:<2s
            a G- t
            , ^<
            @      2