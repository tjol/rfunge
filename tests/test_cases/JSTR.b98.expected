test 1 is a success
as is test 2
